titus-client/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/GrpcJobManagementModelConverters.java

```diff
@@ -371,6 +374,24 @@ private static BasicContainer toCoreBasicContainer(com.netflix.titus.grpc.protog
                 .build();
     }
 
+    private static List<Volume> toCoreVolumes(List<com.netflix.titus.grpc.protogen.Volume> volumes) {
+        return volumes.stream().map(GrpcJobManagementModelConverters::toCoreVolume).collect(Collectors.toList());
```

Comment:
We should avoid Java stream API usage, as it is very slow and produces a lot of garbage in the heap.

Reply:
Can you recommend a preferred pattern for `map` over an array like this?

Reply:
We should use `for` loop in such case.

Reply:
Ok easy. Let me know if you would like me to make a second PR on this file to replace all the stream() usage to for loops too.